text/0227-separate-semantic-conventions.md

```diff
@@ -0,0 +1,179 @@
+convention would require bumping the version number of the entirety of the
+OpenTelemetry specification.
+
+## Explanation
+
+A new github repository called `semantic-conventions` would be created in the
+OpenTelemetry organization.
+
+This would *initially* have the following structure:
+
+- Boilerplate files, e.g. `README.md`, `LICENSE`, `CODEOWNERS`, `CONTRIBUTING.md`
+- `Makefile` that allows automatic generation of documentation from model.
+- `semantic_conventions/` The set of YAML files that exist in
+  `{specification}/semantic_conventions` today.
+- `docs/` A new directory that contains human readable documentation for how to
+  create instrumentation compliant with semantic conventions.
+  - `resource/` - Contents of `{specification}/resource/semantic_conventions`
+  - `trace/` - Contents of `{specification}/trace/semantic_conventions`
+  - `metrics/` - Contents of `{specification}/metrics/semantic_conventions`
+  - `logs/`- Contents of `{specification}/logs/semantic_conventions`
+  - `schemas/` - A new location for [Telemetry Schemas](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/schemas/README.md)
+    to live. This directory will be hosted at
+    `https://opentelemetry.io/schemas/`
+
+Existing semantic conventions in the specification would be marked as
+moved, with documentation denoting the move, but preserving previous contents.
+
+Additionally, if the semantic conventions eventually move to domain-specific
+directory structure (e.g. `docs/{domain}/README.md`, with trace, metrics, events
+in the same file), then this can be refactored in the new repository, preserving
+git history.
+
+There will also be the following exceptions in the specification:
+
+- Semantic conventions used to implement API/SDK details will be fully specified in the `opentelemetry-specification` repo
+  and will not be allowed to change in the Semantic Convention directory.
+  - Error/Exception handling will remain in the specification.
+  - SDK configuration interaction w/ semantic convention will remain in the
+    specification. Specifically `service.name`.
+- The specification may elevate some semantic conventions as necessary for
+  compatibility requirements, e.g. `service.instance.id` and
+  [Prometheus Compatibility](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md).
+
+These exceptions exist because:
+
+- Stable portions of the specification already rely on these conventions.
+- These conventions are required to implement an SDK today.
+
+As such, the Specification should define the absolute minimum of reserved or
+required attribute names and their interaction with the SDK.
+
+## Internal details
+
+The following process would be used to ensure semantic conventions are
+seamlessly moved to their new location. This process lists steps in order:
+
+- A moratorium will be placed on Semantic Convention PRs to the specififcation
```

Comment:
if HTTP semconv PRs are going to be blocked until semconv 2.0 is released from the new repo, I'm interested to see language that this move is going to start (at least) soon

```suggestion
- As soon as this OTEP is merged,
- a moratorium will be placed on Semantic Convention PRs to the specififcation
```

Reply:
OTEPs go into effect when they are merged (unless stated otherwise)

```suggestion
- A moratorium is placed on Semantic Convention PRs to the specififcation
```

Reply:
specififcation -> specification

Reply:
This isn't how OTEPs work.  OTEPs are design docs, plans.  The actually changes need to hit the Specification and go through approval there.

SO yes, this OTEP (the plan) goes into effect when merged.  However, a moratorium on the Specificiation still needs to be a PR *in* the specification repository with clear messaging to specification contributors.  While most of us may follow OTEPS + specs, not everyone does.  This is why the execution of the plan hinges on that PR.